tests/acceptance/config/behat.yml

```diff
@@ -211,6 +211,26 @@ default:
         - OcisConfigContext:
         - PublicWebDavContext:
 
+    apiFullTextSearch:
+      paths:
+        - '%paths.base%/../features/apiFullTextSearch'
+      context: *common_ldap_suite_context
+      contexts:
+        - SettingsContext:
+        - GraphContext:
+        - SpacesContext:
+        - PublicWebDavContext:
+        - FeatureContext: *common_feature_context_params
+        - SearchContext:
+        - CapabilitiesContext:
+        - FilesVersionsContext:
+        - OCSContext:
+        - TrashbinContext:
+        - PublicWebDavContext:
+        - WebDavPropertiesContext:
+        - FavoritesContext:
+        - ChecksumContext:
```

Comment:
```suggestion

```
This won't be needed after backport of this PR https://github.com/owncloud/ocis/pull/6627 is done. So waiting till make worth.

Reply:
Okay